jerry-core/include/jerryscript-port.h

```diff
@@ -96,7 +98,7 @@ typedef enum
  * Example: a libc-based port may implement this with vfprintf(stderr) or
  * vfprintf(logfile), or both, depending on log level.
  */
-void jerry_port_log (jerry_log_level_t level, const char *format, ...) __attribute__ ((format (printf, 2, 3)));
+void JERRY_ATTR_FORMAT (printf, 2, 3) jerry_port_log (jerry_log_level_t level, const char *format, ...);
```

Comment:
jerry_port_log new line?

Reply:
Please explain, I'm not sure I know what you mean.

Reply:
```
JERRY_ATTR_FORMAT (printf, 2, 3) 
```
Are to long in a single line with jerry_port_log (jerry_log_level_t level, const char *format, ...);

Are the following style acceptble?
```c
void JERRY_ATTR_FORMAT (printf, 2, 3)
jerry_port_log (jerry_log_level_t level, const char *format, ...);

```

Reply:
That 's a possible way to write it but our line length limits are 120 chars. In the headers, we usually keep one declaration in one line if the limit allows, so I see no point in changing the style here (especially because it was written in a single line before the change).